packages/internal-lib-build/bin/internal-lib-build.js

```diff
@@ -31,9 +31,9 @@ const main = () => {
         sh.rm('-rf', './dist')
         sh.mkdir('./dist')
         execSync(
-            `${babel} --config-file ${babelConfig} src -x ".js",".jsx",".ts",".tsx" --ignore "**/test_fixtures/*","*.test.js","test.js" --out-dir dist --copy-files`
+            `"${babel}" --config-file "${babelConfig}" src -x ".js",".jsx",".ts",".tsx" --ignore "**/test_fixtures/*","*.test.js","test.js" --out-dir dist --copy-files`
```

Comment:
Any place that you see me adding doube-quotes around executables it's because I'm doing it to support home paths that have spaces in them. This is common on windows.